script/bootstrap

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Sets up all the dependencies, etc. so you can build and test.
+
+set -euo pipefail
+source "$(dirname "$0")/utilities.bash"
+
+header "Fetching utilites"
+go get "$@" -v \
+  github.com/alecthomas/gometalinter \
+  github.com/mitchellh/gox \
+  github.com/kardianos/govendor \
+  github.com/wadey/gocovmerge \
+  github.com/pengwynn/flint
+
```

Comment:
You should probably update the above list. For example, you seem to using `glide`, not `govendor`.